Modules/xSharePoint/DSCResources/MSFT_xSPOutgoingEmailSettings/MSFT_xSPOutgoingEmailSettings.psm1

```diff
@@ -0,0 +1,86 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $WebAppUrl,
+        [parameter(Mandatory = $true)] [System.String] $SMTPServer,
+        [parameter(Mandatory = $true)] [System.String] $FromAddress,
+        [parameter(Mandatory = $true)] [System.String] $ReplyToAddress,
+        [parameter(Mandatory = $true)] [System.String] $CharacterSet
+        
+    )
+
+    Write-Verbose -Message "Retrieving outgoing email settings configuration "
+
+    $result = Invoke-xSharePointCommand -Arguments $PSBoundParameters -ScriptBlock {
```

Comment:
I thought you had told me not to have the installaccount. Easy enough. I will re-add it:)


> On 17 Nov 2015, at 8:16 AM, Brian Farnhill <notifications@github.com> wrote:
> 
> In Modules/xSharePoint/DSCResources/MSFT_xSPOutgoingEmailSettings/MSFT_xSPOutgoingEmailSettings.psm1:
> 
> > +{
> > +    [CmdletBinding()]
> > +    [OutputType([System.Collections.Hashtable])]
> > +    param
> > +    (
> > +        [parameter(Mandatory = $true)] [System.String] $WebAppUrl,
> > +        [parameter(Mandatory = $true)] [System.String] $SMTPServer,
> > +        [parameter(Mandatory = $true)] [System.String] $FromAddress,
> > +        [parameter(Mandatory = $true)] [System.String] $ReplyToAddress,
> > +        [parameter(Mandatory = $true)] [System.String] $CharacterSet
> > +        
> > +    )
> > +
> > +    Write-Verbose -Message "Retrieving outgoing email settings configuration "
> > +
> > +    $result = Invoke-xSharePointCommand -Arguments $PSBoundParameters -ScriptBlock {
> You need to add an InstallAccount parameter (look at pretty much any other resource we have to see how we do it), this is needed to ensure we can support PowerShell 4 where PsDscRunAsCredential is not available.
> 
> —
> Reply to this email directly or view it on GitHub.
>